scripts/Muon/GUI/ElementalAnalysis/peak_data.json

```diff
@@ -3,2370 +3,2463 @@
+      "N(8g->4f)": 801.2,
+      "N(8g->4f) ": 809.9,
+      "O(7h->5g)": 373.1
+    },
+    "Gammas": {
+      "g205Tl(100) ": 203.7,
+      "g205Tl(100)  ": 415.7,
+      "g205Tl(100)   ": 720.1,
+      "g205Tl(100)    ":937.1,
+      "g205Tl(100)     ":976.3,
+      "g205Tl(100)      ": 1218.7,
+      "g205Tl(100)       ": 1433.9,
+      "g205Tl(100)        ": 1234.8,
+      "g206Tl(100)": 265.8,
+      "g206Tl(100) ": 304.9,
+      "g206Tl(100)  ": 330.1,
+      "g206Tl(100)   ": 535.5,
+      "g206Tl(100)    ": 851.4,
+      "g206Tl(100)     ": 1055.2,
+      "g206Tl(100)      ": 453.2,
+      "g206Tl(100)       ": 686.3,
+      "g206Tl(100)        ": 732.3,
+      "g206Tl(100)         ": 278.2
+
+
     },
     "Electrons": {
-      "72.8": 0.59,
-      "75.0": 1,
-      "84.45": 0.12,
-      "84.94": 0.22,
-      "87.3": 0.1
+    "0.59": 72.8,
+    "1": 75.0,
+    "0.12": 84.45,
+    "0.22": 84.94,
+    "0.1": 87.3
     }
   },
   "Pd": {
     "Z": 46,
     "A": 106.42,
     "Primary": {
-      "K(7->1)": 3087.6,
-      "L(3->2)": 833.6,
-      "L(4->2)": 864.2,
-      "M(4->3)": 291.6,
-      "5->4": 134.2
-    },
-    "Secondary": {
-      "K(2->1)": 3039.2,
-      "K(3->1)": 3045.4,
-      "K(4->1)": 3054.1,
-      "K(5->1)": 3074,
-      "K(6->1)": 3081.4,
-      "K(8->1)": 3095.1,
-      "K(9->1)": 3914.8,
-      "L(5->2)": 1124.9,
-      "L(6->2)": 1158,
-      "L(7->2)": 1260,
-      "L(8->2)": 1293.9,
-      "M(5->3)": 295,
-      "M(6->3)": 426.2,
-      "M(7->3)": 429.9,
-      "M(8->3)": 499.1,
-      "M(9->3)": 502.8,
-      "M(10->3)": 542.4,
-      "M(11->3)": 546.6,
-      "M(12->3)": 591.6,
-      "6->4": 207.4,
-      "7->4": 251.6
-    }
-  },
-  "Pm": {
-    "Z": 61,
-    "A": null,
-    "Primary": null,
-    "Secondary": null
+      "K(2p->1s)": 3087.6,
+      "L(3d->2p)": 833.6,
+      "L(3d->2p) ": 864.2,
+      "M(4f->3d)": 291.6,
+      "M(4f->3d) ": 295,
+      "N(5g->4f)": 134.2
+    },
+    "Secondary": {
+      "K(2p->1s)": 3039.2,
+      "K(2p->1s) ": 3045.4,
+      "K(2p->1s)  ": 3054.1,
+      "K(2p->1s)   ": 3074,
+      "K(2p->1s)    ": 3081.4,
+      "K(2p->1s)     ": 3095.1,
+      "K(3p->1s)": 3914.8,
+      "L(4d->2p)": 1124.9,
+      "L(4d->2p) ": 1158,
+      "L(5d->2p)": 1260,
+      "L(5d->2p) ": 1293.9,
+      "M(5f->3d)": 426.2,
+      "M(5f->3d) ": 429.9,
+      "M(6f->3d)": 499.1,
+      "M(6f->3d) ": 502.8,
+      "M(7f->3d)": 542.4,
+      "M(7f->3d) ": 546.6,
+      "M(8f->3d)": 591.6,
+      "M(8f->3d) ": 595.8,
+      "N(6g->4f)": 207.4,
+      "N(7g->4f)": 251.6
+    }
   },
   "Pr": {
     "Z": 59,
     "A": 140.91,
     "Primary": {
-      "K(2->1)": 4185.9,
-      "K(3->1)": 4263.6,
-      "L(3->2)": 1359.1,
-      "M(4->3)": 482,
-      "M(5->3)": 491.3,
-      "5->4": 222.7
-    },
-    "Secondary": {
-      "K(4->1)": 5610.1,
-      "L(4->2)": 1425.3,
-      "L(5->2)": 1839.8,
-      "L(6->2)": 1912.3,
-      "L(7->2)": 2061.8,
-      "L(8->2)": 2131.4,
-      "M(6->3)": 703,
-     
```

Comment:
is the `~` intential?

Reply:
I assume it is. It could signify an approximation